terraform/stacks/umccr_datalake/policies/datalake_role_inline_policy.json

```diff
@@ -0,0 +1,33 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetObject",
+        "s3:PutObject"
+      ],
+      "Resource": [
+        "arn:aws:s3:::${bucket}/*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "lakeformation:*"
```

Comment:
Nice, I assume this policy would've helped the other day when I had to try all sorts of lake formation permission combinations to get through these errors: https://aws.amazon.com/premiumsupport/knowledge-center/glue-insufficient-lakeformation-permissions/

Reply:
Good pointer to the article, Pete. TBH; I just copy in this policy from the Glue role that auto setup through Glue Console. Let me read up the article and, see if I can fine tune a bit. Looks like we could/should tune a bit here, yes.

Reply:
Right! Done a bit of reading. And removed LakeFormation permission with https://github.com/umccr/infrastructure/commit/d7bf0859418e9e506d3ef578d2ba84cca4bc2424

* Turn out we do not need LakeFormation permission just yet as we can just use Glue default IAM-based permission.
* We can upgrade to LakeFormation access control model in future when it sees fit for more complex access control use cases.
* For that, see
https://docs.aws.amazon.com/lake-formation/latest/dg/upgrade-glue-lake-formation.html